content/en/docs/reference/kubernetes-api/authentication-resources/service-account-v1.md

```diff
@@ -16,7 +16,10 @@ weight: 1
 
 ## ServiceAccount {#ServiceAccount}
 
-ServiceAccount binds together: * a name, understood by users, and perhaps by peripheral systems, for an identity * a principal that can be authenticated and authorized * a set of secrets
+ServiceAccount binds together: 
```

Comment:
Hello @codedevote .
This content is auto-generated by the API reference generator.
/cc @feloy .